gradle-plugin/it/src/it/probes-groovy-dsl-config/src/main/java/org/eclipse/jkube/it/gradle/probes/groovy/dsl/probesgroovydslconfig/ProbesGroovyDslConfigApplication.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.it.gradle.probes.groovy.dsl.probesgroovydslconfig;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ProbesGroovyDslConfigApplication {
+
+	public static void main(String[] args) {
```

Comment:
If removing the actuator dependency, very simple GetMapping/RequestMapping endpoints could be created.

```java
@GetMapping(value = "/liveness/startup")
@ResponseBody
public String getStartup() {
    return "OK";
}
// Same for liveness, readiness
```